### PR TITLE
client_endpoints: Create new client endpoints and demo fio benchmark.

### PR DESCRIPTION
**Pull request by markhpc**
Signed-off-by: Mark Nelson <mnelson@redhat.com>

### CODE REVIEW

**Comment by markhpc** [comment]
- Example cbt yaml file: https://paste.fedoraproject.org/paste/a1y9wHFo58MNs49v1g3qZw
          - Initial test results: https://docs.google.com/spreadsheets/d/1oJZ036QDbJQgv2gXts1oKKhMOKXrOI2XLTkvlsl9bUs/edit?usp=sharing

File: client_endpoints/cephfsfuse_client_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class CephfsFuseClientEndpoints(CephClientEndpoints):
+    def create(self):
+        self.create_fs()
+
+    def mount(self):
+        self.mount_fs()
+
+    def mount_fs_helper(self, node, dir_name):
+        cmd = 'sudo mount -t ceph -o mds_namespace=%s %s %s' % (self.name, ','.join(self.mon_addrs), dir_name)
```
Comment (batrick) [inline client_endpoints/cephfsfuse_client_endpoints.py]: Is this supposed to mount with `ceph-fuse`?

File: client_endpoints/cephfskernel_client_endpoints.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+class CephfsKernelClientEndpoints(CephClientEndpoints):
+    def create(self):
+        self.create_fs()
+
+    def mount(self):
+        self.mount_fs()
+
+    def mount_fs_helper(self, node, dir_name):
+        cmd = 'sudo mount -t ceph -o mds_namespace=%s %s %s' % (self.name, ','.join(self.mon_addrs), dir_name)
```
Comment (batrick) [inline client_endpoints/cephfskernel_client_endpoints.py]: @ukernel any other options for performance testing?

File: client_endpoints/cephfsfuse_client_endpoints.py (hunk 1 above)
Comment (markhpc) [reply client_endpoints/cephfsfuse_client_endpoints.py]: Ugh!  Apparently when I was moving stuff around I must have overwritten it with the kernel version.  I'm testing a fix now.  That also probably explains why the results here look so similar to the kernel version:

https://docs.google.com/spreadsheets/d/1oJZ036QDbJQgv2gXts1oKKhMOKXrOI2XLTkvlsl9bUs/edit?usp=sharing

I'll need to update those.

**Comment by markhpc** [comment]
Added the example as a separate commit, merging this now.

**Pull request merged**